Source/rclUE/Public/Srvs/ROS2SpawnEntitiesSrv.h

```diff
@@ -0,0 +1,271 @@
+            }
+
+            out_ros_data.state.data[i].pose.position.x = PositionList[i].X;
+            out_ros_data.state.data[i].pose.position.y = PositionList[i].Y;
+            out_ros_data.state.data[i].pose.position.z = PositionList[i].Z;
+
+            out_ros_data.state.data[i].pose.orientation.x = OrientationList[i].X;
+            out_ros_data.state.data[i].pose.orientation.y = OrientationList[i].Y;
+            out_ros_data.state.data[i].pose.orientation.z = OrientationList[i].Z;
+            out_ros_data.state.data[i].pose.orientation.w = OrientationList[i].W;
+
+            out_ros_data.state.data[i].twist.linear.x = TwistLinearList[i].X;
+            out_ros_data.state.data[i].twist.linear.y = TwistLinearList[i].Y;
+            out_ros_data.state.data[i].twist.linear.z = TwistLinearList[i].Z;
+            out_ros_data.state.data[i].twist.angular.x = TwistAngularList[i].X;
+            out_ros_data.state.data[i].twist.angular.y = TwistAngularList[i].Y;
+            out_ros_data.state.data[i].twist.angular.z = TwistAngularList[i].Z;
+
+            {
+                FTCHARToUTF8 strUtf8(*ReferenceFrameList[i]);
+                int32 strLength = strUtf8.Length();
+                out_ros_data.state.data[i].reference_frame.data = (decltype(out_ros_data.state.data[i].reference_frame.data))malloc(
+                    (strLength + 1) * sizeof(decltype(*out_ros_data.state.data[i].reference_frame.data)));
+                memcpy(out_ros_data.state.data[i].reference_frame.data,
+                       TCHAR_TO_UTF8(*ReferenceFrameList[i]),
+                       (strLength + 1) * sizeof(char));
+                out_ros_data.state.data[i].reference_frame.size = strLength;
+                out_ros_data.state.data[i].reference_frame.capacity = strLength + 1;
+            }
+        }
+        if (out_ros_data.tags.data != nullptr)
+        {
+            free(out_ros_data.tags.data);
+        }
+        out_ros_data.tags.data =
+            (decltype(out_ros_data.tags.data))malloc((TagsList.Num()) * sizeof(decltype(*out_ros_data.tags.data)));
+        for (int i = 0; i < TagsList.Num(); i++)
+        {
+            {
+                FTCHARToUTF8 strUtf8(*TagsList[i]);
+                int32 strLength = strUtf8.Length();
+                if (out_ros_data.tags.data[i].data != nullptr)
+                {
+                    free(out_ros_data.tags.data[i].data);
+                }
+                out_ros_data.tags.data[i].data = (char*)malloc((strLength + 1) * sizeof(char));
+                memcpy(out_ros_data.tags.data[i].data, TCHAR_TO_UTF8(*TagsList[i]), (strLength + 1) * sizeof(char));
+                out_ros_data.tags.data[i].size = strLength;
+                out_ros_data.tags.data[i].capacity = strLength + 1;
+            }
+        }
+    }
+};
+
+USTRUCT(Blueprintable)
+struct RCLUE_API FROSSpawnEntitiesResponse
+{
+    GENERATED_BODY()
+
+public:
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    bool bSuccess;
```

Comment:
```suggestion
    bool bSuccess = False; 
```
as by-default set [here](https://github.com/rapyuta-robotics/UE_tools/blob/devel/CodeGen/CodeGeneratorFromPath.py#L606)?